### DISCUSSION TITLE
Event Viewer Logs

### DISCUSSION THREAD

**Discussion by kkamran1010** [SecurityRiskAdvisors/VECTR#245] [Q&A] [answered]
Looking to import the Access/security events into a SIEM tool. Where can i find that path that those logs live?

**Comment by SRAPSpencer** [comment, marked as answer]
Auth isn't currently accessible via the API. 

I'll mark this as a feature request.

**Reply by kkamran1010** [reply]
is there a log file somewhere that can just be viewed locally on the server?

**Reply by SRAPSpencer** [reply]
Any logs written to a disk file would be inside the container under 

usr/local/tomcat/logs

There are some, but I believe the logs you're looking for only exist in the DB

**Reply by doodleincode** [reply]
There is a log that captures basic HTTP requests to the auth API, however it's not "enriched" with identity data (other than remote IP). The logs are available in the tomcat container in `/usr/local/tomcat/logs`. Files are `vectr_audit-YYYY-MM-DD-<n>.log`
